integration_tests/test_domestic_migration.py

```diff
@@ -123,3 +123,88 @@ def test_only_living_people_move(simulants_on_adjacent_timesteps):
         movers = before["address_id"] != after["address_id"]
         assert after[movers]["tracked"].all()
         assert (after[movers]["alive"] == "alive").all()
+
+
+# add employer address tests
+# 2. all simulants with same employer id have same employer details (static)
+# 3. all employer ids have unique address details (static)
+
+
+def test_addresses_during_moves(simulants_on_adjacent_timesteps):
+    """Check that address details change after a move. Note that
+    this test does not distinguish between the different types of moves (eg
+    household vs individual moves)
+    """
+    unit_id_col = "employer_id"
+    address_id_col = "business_details.employer_address_id"
+    other_details_cols = ["business_details.employer_name"]
+    _test_addresses_during_moves(
```

Comment:
This is easier to read if you just put the strings directly in the function call rather than define them in variables above.